### PR TITLE
Jenkinsimage reference jenkinsimagephase

### PR DESCRIPTION
**Pull request by akram**

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -3,10 +3,10 @@ run:
 linters-settings:
   errcheck:
     check-blank: false
-    ignore: fmt:.*,io/ioutil:^Read.*,Write
+    ignore: fmt:.*,io/ioutil:^Read.*,Write,zz_generated*
 
 linters:
-  enable-all: true
+  enable-all: false
```
Comment (jkhelil) [inline .golangci.yml]: are we disabling linting ?

Comment (akram) [reply .golangci.yml]: this parameter is deprecated, I will use the disable-all + enable approach.

**Comment by waveywaves** [comment]
Just reviewed the PR, this looks alright

**Pull request merged**